### PR TITLE
Cyberark/migrate slosilo gem

### PR DESCRIPTION
**Pull request by jeniaSakirko**
### Desired Outcome

Conjur is currently the only project which uses the slosilo gem. This PR brings it into Conjur to simplify the development.

### Implemented Changes

This commit adds `slosilo` as a local gem.

### Connected Issue/Story

N/A - This is being done in support of Conjur SaaS.

CyberArk internal issue ID: [insert issue ID]

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [x] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]
- [x] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [x] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [x] There are no security aspects to these changes

### CODE REVIEW

**Review by sofiadimant1 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -73,4 +73,24 @@ conjur_git_commit
...
+gem/slosilo/rdoc
+gem/slosilo/spec/reports
+gem/slosilo/test/tmp
+gem/slosilo/test/version_tmp
+gem/slosilo/tmp
+gem/slosilo/.rvmrc
+gem/slosilo/.project
+gem/slosilo/.kateproject.d
+gem/slosilo/.idea
+
```
Comment (jvanderhoof) [inline .gitignore:95]: Why are we ignoring all the Slosilo files?

File: gems/slosilo/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# v3.0.1
```
Comment (jvanderhoof) [inline gems/slosilo/CHANGELOG.md:1]: Mind adding a note to the top of this file to direct users to add Slosilo changes to the main `Changelog.md`?

Example from Conjur-Rack: https://github.com/cyberark/conjur/blob/master/gems/conjur-rack/CHANGELOG.md

File: .gitignore (hunk 1 above)
Comment (sofiadimant1) [reply .gitignore:95]: these are the ignored files frm .gitignore file the was in slosilo library, 
these are not all the slosilo files..

File: gems/slosilo/CHANGELOG.md (hunk 2 above)
Comment (jeniaSakirko) [reply gems/slosilo/CHANGELOG.md:4]: Done

**Pull request closed without merging**